.gitlab/ci/bucket-upload.yml

```diff
@@ -456,3 +457,17 @@ slack-notify-bucket-upload:
   extends:
     - .trigger-slack-notification
     - .bucket-upload-rule-always
+
+
+upload-id-set-bucket:
```

Comment:
Will this job be required for passsing the build?

Reply:
This will be required to pass the upload-flow build.
We would like to know if it fails, because other builds and the demisto-sdk still download the official XSOAR id-set from the marketplace.
When we will fully migrate to the content graph, this job will be removed